cmd/flowrunner/testdata/flows/two_questions.json

```diff
@@ -200,7 +200,7 @@
                             "type": "call_webhook",
                             "url": "http://localhost/?cmd=success",
                             "method": "POST",
-                            "body": "{ \"contact\": @(to_json(contact.uuid)), \"soda\": @(to_json(run.results.soda)) }"
+                            "body": "{ \"contact\": @(to_json(contact.uuid)), \"soda\": @(to_json(run.results.soda.value)) }"
```

Comment:
this is so that we still just get the value - `@(to_json(run.results.soda))` includes category etc